### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ndoschek**
I updated to the latest modelserver nightly and aligned some other dependencies as well.
Now the timeout functionality should also work as expected in ecore-glsp.

Signed-off-by: Nina Doschek <ndoschek@eclipsesource.com>

### CODE REVIEW

File: server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/model/EcoreModelServerAccess.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ public void update(GModelState graphicalModelState) {
 	public void subscribe(NotificationSubscriptionListener<EObject> subscriptionListener) {
 		LOGGER.debug("EcoreModelServerAccess - subscribe");
 		this.subscriptionListener = subscriptionListener;
-		this.modelServerClient.subscribe(modelUri, subscriptionListener);
+		this.modelServerClient.subscribe(modelUri, subscriptionListener, "xmi");
```
Comment (CamilleLetavernier) [inline server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/model/EcoreModelServerAccess.java:84]: You can use the FORMAT_XMI constant instead

**Review by CamilleLetavernier — APPROVED** [review]
Thanks Nina, the changes look good to me! Just one minor comment

File: server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/model/EcoreModelServerAccess.java (hunk 1 above)
Comment (ndoschek) [reply server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/model/EcoreModelServerAccess.java:84]: Sure, thanks for spotting!

**Review by CamilleLetavernier — APPROVED** [review]
Looks good to me, thank you!

**Pull request merged**